Gemfile

```diff
@@ -1,4 +1,2 @@
 source 'https://rubygems.org'
 gemspec
-
-gem 'redis'
```

Comment:
FWIW this is how you'd do a soft dependency for this repo. It'd be a hard dependency if it were added to the `.gemspec` file instead as a `runtime_dependency`.